docs/examples/sglang/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
you can put this in the same lws.yaml file

Reply:
Thanks @ahg-g 
It has been changed :-)